cmd/minishift/cmd/addon/addon_install.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 const (
-	unspecifiedSourceError   = "You need to specify the source for the add-on."
+	unspecifiedSourceError   = "You must specify the source directory path of the add-on."
```

Comment:
It intentionally only says source here without further qualifications. We are about to implement other source types. "You must specify the source of the add-on."